docs/local-development.md

```diff
@@ -27,6 +27,11 @@ To use [Drupal VM](http://www.drupalvm.com/) with a Drupal project that is gener
 1. Create & boot the VM
 
         blt vm
+        
+1. Commit all resulting files, including `box/config.yml` and `Vagrantfile`.
+        
+        git add -A
+        git commit -m "Added DrupalVM."
```

Comment:
This commit message wouldn't pass the pre-commit hook's check for project initials and ticket number, I think.